### PR TITLE
Create TAC-Decision-Process.md

### PR DESCRIPTION
**Pull request by SecurityCRob**
proposed documentation for how the TAC should make decisions

### CODE REVIEW

File: process/TAC-Decision-Process.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# TAC Decision Process #
+
+The OpenSSF Technical Advisory Council (TAC) is a body comprised of 9 individuals that are either elected for a 2 year term or appointed by the governing board for a 1 year term.
```
Comment (mlieberman85) [inline process/TAC-Decision-Process.md:3]: I thought it was split where some elected folks were also 1 year terms. I don't remember if we changed this to all elected members get 2 year terms.

File: process/TAC-Decision-Process.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Broadly, there are three main methods that the TAC will use to make decisions and ensure those decisions are appropriately documented:
+
+## 1.) Simple and/or non-consequential changes ##
+For asynchronous work items that do not have long-term/strategic consequences, the TAC will use lazy consensus to determine whether to adopt the proposal and ensure that it will be properly documented.  Issues and PRs should be created within the TAC GitHub repository and assigned to the TAC group for review.  Provided that at least two TAC members agree, support, and approve the suggested change, there are no objections raised by any TAC members, and no less than one week has transpired since the TAC was notified of the change, the proposal will be merged and adopted.
```
Comment (mlieberman85) [inline process/TAC-Decision-Process.md:12]: Can we include an example of what does have long-term/strategic consequences? I think "adoption of a new TI" maybe is long term?

File: process/TAC-Decision-Process.md (hunk 1 above)
Comment (SecurityCRob) [reply process/TAC-Decision-Process.md:3]: Everyone elected will have a 2year term going forward.  For this first year, where we started the staggering or terms, we asked for volunteers to have a 1year term (and the folks that had served the prior year agreed for this.  Ideally we will capture this in Arnaud's forthcoming awesome TAC election process docs.

File: process/TAC-Decision-Process.md (hunk 2 above)
Comment (SecurityCRob) [reply process/TAC-Decision-Process.md:12]: we sure can! please feel free to suggest something

**Review by marcelamelara — COMMENTED** [review]
Thanks for getting this started @SecurityCRob !

File: process/TAC-Decision-Process.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+This includes such things as documentation updates/minor changes
```
[the remark this answers is not in the document]
Comment (SecurityCRob) [reply process/TAC-Decision-Process.md:14]: a bit, yes.  me--

**Comment by marcelamelara** [comment]
To make the decision process even clearer, I think we could look at how SLSA handles [different PR types](https://github.com/slsa-framework/slsa/blob/main/CONTRIBUTING.md#pull-request-types). Not suggesting that those types all make direct sense in this context, but I like how the table concisely enumerates decision categories, decision timelines/periods, and number of needed approvers for each. So because this doc is aiming to codify different decision categories for us, I wonder if we'd want to adopt a similar format for this doc? What do others think?

**Comment by SecurityCRob** [comment]
> To make the decision process even clearer, I think we could look at how SLSA handles [different PR types](https://github.com/slsa-framework/slsa/blob/main/CONTRIBUTING.md#pull-request-types). Not suggesting that those types all make direct sense in our context, but I like how the table concisely enumerates decision categories, decision timelines/periods, and number of needed approvers for each. So because this doc is aiming to codify different decision categories for us, I wonder if we'd want to adopt a similar format for this doc? What do others think?

<3  I'll see what I can do to Goose-ify that table.  tyvm!!!!!

File: process/TAC-Decision-Process.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+## 2.) For work items discussed at the periodic TAC calls ##
+The TAC must have quorum (majority of members present) and a simple majority of the members present must approve of a proposal or topic.  This decision will be captured in the TAC meeting notes as well as an issue within the TAC repository. 
+
+
+
+## 3.) For  major items that require a formal vote ##
+An issue or PR must be created within the TAC repository.  TAC members will be asked via the TAC mailing list to review, comment, and decline/approve the item within the GitHub issue/PR.  The TAC will be given 10 business days to review and consider the request.  These topics ideally should also be presented at a TAC meeting and publicly discussed/debated prior to the closing of voting.  
+
+This includes items such as changes to the TAC technical vision, Foundation strategy, wg/project adoption, funding requests, etc.
```
Comment (lehors) [inline process/TAC-Decision-Process.md:25]:
```suggestion
This includes items such as changes to the TAC technical vision, Foundation strategy, WG/project adoption, funding requests, etc.
```

File: process/TAC-Decision-Process.md (hunk 2 above)
Comment (lehors) [reply process/TAC-Decision-Process.md:12]: Isn't that what line 14 is trying to do?

File: process/TAC-Decision-Process.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+Every PR must be categorized using one of the following <type> values. The purpose is twofold: to make it easier for readers to understand the scope of the PR at a glance, and to allow us to adjust the minimum review period and number of approvers based on how sensitive the PR is.
+
+Use the closest entry in the table that applies, selecting the first one if multiple apply. If you are not sure which type to use, take a guess and a maintainer will update if needed. See review and approval for the meaning of "waiting period" and "# approvers".
+
+|  Type  | Meaning | Waiting period |	# Approvers |
+| ------ | ------- | -------------- | ----------- |
+| content |	A change to the meaning of the specification. Must include a changelog entry. |	72h |	3 |
+| editorial |	A clarification to the specification that does not change its meaning, beyond a simple fix. |	24h |	2 |
+| nonspec | 	A change to a non-specification, non-blog page, beyond a simple fix. |	24h |	2 |
+| blog | 	A new or updated blog post. (Do not mix with categories above.) | 	24h |	2 |
```
Comment (marcelamelara) [inline process/TAC-Decision-Process.md:38]: Should this also cover tech talks?

**Review by marcelamelara — COMMENTED** [review]
Thanks for adding the table! One type of review I haven't seen mentioned yet are for urgent comms, should that be added here?

**Review by sevansdell — APPROVED** [review]
I think this is a good start and we can keep iterating.

File: process/TAC-Decision-Process.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+Broadly, there are three main methods that the TAC will use to make decisions and ensure those decisions are appropriately documented:
+
+## 1.) Simple and/or non-consequential changes ##
+For asynchronous work items that do not have long-term/strategic consequences, the TAC will use lazy consensus to determine whether to adopt the proposal and ensure that it will be properly documented.  Issues and PRs should be created within the TAC GitHub repository and assigned to the TAC group for review.  Provided that at least two TAC members agree, support, and approve the suggested change, there are no objections raised by any TAC members, and no less than one week has transpired since the TAC was notified of the change, the proposal will be merged and adopted.
+
+This includes such things as documentation updates/minor changes
+
+
+## 2.) For work items discussed at the periodic TAC calls ##
```
[the remark this answers is not in the document]
Comment (SecurityCRob) [reply process/TAC-Decision-Process.md:17]: does the addition of "new wg" to "major" address this or do we need to write more?

File: process/TAC-Decision-Process.md (hunk 5 above)
Comment (SecurityCRob) [reply process/TAC-Decision-Process.md:39]: we could, where do you see those fitting into the table or upper sections?

File: process/TAC-Decision-Process.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+# Pull request types #
+
+Every PR must be categorized using one of the following <type> values. The purpose is twofold: to make it easier for readers to understand the scope of the PR at a glance, and to allow us to adjust the minimum review period and number of approvers based on how sensitive the PR is.
+
+Use the closest entry in the table that applies, selecting the first one if multiple apply. If you are not sure which type to use, take a guess and a maintainer will update if needed.
+
+|  Type  | Meaning | Waiting period |	# Approvers |
+| ------ | ------- | -------------- | ----------- |
+| TI lifecycle | A new TI or updated TI lifecycle stage. | 10d | 7 |
```
Comment (steiza) [inline process/TAC-Decision-Process.md:35]: I've been assuming this was 5 (a majority) not 7 (a supermajority) - I just want to clarify if you intended a majority or supermajority.

**Review by mlieberman85 — APPROVED** [review]
Looking good

File: process/TAC-Decision-Process.md (hunk 7 above)
Comment (SecurityCRob) [reply process/TAC-Decision-Process.md:35]: Interesting.  Hmm... we probably need a category for each scenario (majority vs. supermajority).  I think simple majority is good for what is listed, but perhaps things like charter changes or technical direction changes need the super?

Comment (SecurityCRob) [reply process/TAC-Decision-Process.md]: made proposed changes that we can talk about in the TAC call today

**Review by lehors — APPROVED** [review]

File: process/TAC-Decision-Process.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+# Pull request types #
+
+Every PR must be categorized using one of the following <type> values. The purpose is twofold: to make it easier for readers to understand the scope of the PR at a glance, and to allow us to adjust the minimum review period and number of approvers based on how sensitive the PR is.
+
+Use the closest entry in the table that applies, selecting the first one if multiple apply. If you are not sure which type to use, take a guess and a maintainer will update if needed.
+
+|  Type  | Meaning | Waiting period |	# Approvers |
+| ------ | ------- | -------------- | ----------- |
+| Major  | Proposed changes to Charter, Technical Strategy, or other cross-foundation changing proposal.  If approved, than moved to the Governing committee for review, move to GB. | 15d | 7 |
+| TI lifecycle | A new TI or updated TI lifecycle stage. | 10d | 5 |
```
Comment (bobcallaway) [inline process/TAC-Decision-Process.md:36]: My vote here would be to bump this up to 7.

**Review by SecurityCRob — COMMENTED** [review]
adjusted based on feedback during 14may TAC call

File: process/TAC-Decision-Process.md (hunk 8 above)
Comment (SecurityCRob) [reply process/TAC-Decision-Process.md]: done!

**Pull request merged**